domains/transfer/.dockerignore

```diff
@@ -8,6 +8,7 @@
 **/.toolstarget
 **/.vs
 **/.vscode
+**/.idea
```

Comment:
Why this?

Reply:
To avoid ide specific thing from the awesome jetbrain tools

Reply:
But isn't that for intelliJ Idea?

Reply:
All jetbrains IDE